repositories/repositories.bzl

```diff
@@ -29,74 +36,86 @@ def ros2_repositories():
+        name = "googletest",
+        sha256 = "8ad598c73ad796e0d8280b082cebd82a630d73e73cd3c70057938a6501bba5d7",
+        strip_prefix = "googletest-1.14.0",
+        url = "https://github.com/google/googletest/archive/refs/tags/v1.14.0.tar.gz",
     )
+    _googletest_deps()
 
     maybe(
         http_archive,
-        name = "pybind11",
-        build_file = "@com_github_mvukov_rules_ros2//repositories:pybind11.BUILD.bazel",
-        sha256 = "d475978da0cdc2d43b73f30910786759d593a9d8ee05b1b6846d1eb16c6d2e0c",
-        strip_prefix = "pybind11-2.11.1",
-        urls = ["https://github.com/pybind/pybind11/archive/refs/tags/v2.11.1.tar.gz"],
+        name = "lz4",
+        build_file = "@com_github_mvukov_rules_ros2//repositories:lz4.BUILD.bazel",
+        sha256 = "0b0e3aa07c8c063ddf40b082bdf7e37a1562bda40a0ff5272957f3e987e0e54b",
+        strip_prefix = "lz4-1.9.4",
+        urls = ["https://github.com/lz4/lz4/archive/refs/tags/v1.9.4.tar.gz"],
     )
 
     maybe(
         http_archive,
-        name = "rules_foreign_cc",
-        sha256 = "476303bd0f1b04cc311fc258f1708a5f6ef82d3091e53fd1977fa20383425a6a",
-        strip_prefix = "rules_foreign_cc-0.10.1",
-        url = "https://github.com/bazelbuild/rules_foreign_cc/archive/refs/tags/0.10.1.tar.gz",
+        name = "nlohmann_json",
+        sha256 = "a22461d13119ac5c78f205d3df1db13403e58ce1bb1794edc9313677313f4a9d",
+        urls = ["https://github.com/nlohmann/json/releases/download/v3.11.3/include.zip"],
     )
 
     maybe(
         http_archive,
-        name = "com_google_googletest",
-        sha256 = "8ad598c73ad796e0d8280b082cebd82a630d73e73cd3c70057938a6501bba5d7",
-        strip_prefix = "googletest-1.14.0",
-        url = "https://github.com/google/googletest/archive/refs/tags/v1.14.0.tar.gz",
+        name = "fmt",
+        build_file = "@com_github_mvukov_rules_ros2//repositories:fmt.BUILD.bazel",
+        sha256 = "1250e4cc58bf06ee631567523f48848dc4596133e163f02615c97f78bab6c811",
+        strip_prefix = "fmt-10.2.1",
+        url = "https://github.com/fmtlib/fmt/archive/10.2.1.tar.gz",
+    )
+    maybe(
+        http_archive,
+        name = "spdlog",
+        build_file = "@com_github_mvukov_rules_ros2//repositories:spdlog.BUILD.bazel",
+        sha256 = "4dccf2d10f410c1e2feaff89966bfc49a1abb29ef6f08246335b110e001e09a9",
+        strip_prefix = "spdlog-1.12.0",
+        url = "https://github.com/gabime/spdlog/archive/v1.12.0.tar.gz",
     )
-
-    _googletest_deps()
 
     maybe(
         http_archive,
         name = "tinyxml2",
         build_file = "@com_github_mvukov_rules_ros2//repositories:tinyxml2.BUILD.bazel",
-        sha256 = "3bdf15128ba16686e69bce256cc468e76c7b94ff2c7f391cc5ec09e40bff3839",
-        strip_prefix = "tinyxml2-10.0.0",
-        urls = ["https://github.com/leethomason/tinyxml2/archive/refs/tags/10.0.0.tar.gz"],
+        sha256 = "cc2f1417c308b1f6acc54f88eb70771a0bf65f76282ce5c40e54cfe52952702c",
+        strip_prefix = "tinyxml2-9.0.0",
```

Comment:
No really a reason to downgrade this, or is there? :)

Reply:
The reason is to have the same version in bzlmod and workspace. tinyxml 10 isn't on BCR (yet?) https://registry.bazel.build/modules/tinyxml2

Reply:
added in https://github.com/bazelbuild/bazel-central-registry/pull/1395

Reply:
back at 10 now